src/middlewared/middlewared/plugins/chart_releases_linux/helm.py

```diff
@@ -41,6 +42,10 @@ def helm_action(self, chart_release, chart_path, config, tn_action):
             )
             stderr = cp.communicate()[1]
             if cp.returncode:
-                raise CallError(f'Failed to {tn_action} chart release: {stderr.decode()}')
+                errmsg = stderr.decode()
+                if tn_action == 'upgrade' or tn_action == 'install':
```

Comment:
Is this conditional required?

Reply:
We, @stavros-k and I, wanted to ensure it only gets used for user-facing helm 'commands' (via the middleware).
So we wanted to ensure it doesn't, by accident, gets used to sanitise other errors as well.

Reply:
All of this function's usages are user facing, so let's just remove the conditional please as all of those actions can benefit from the standardisation